steamtinkerlaunch

```diff
@@ -3362,6 +3363,7 @@ function setDefaultCfgValues {
 		if [ -z "$USESPEKD3D47" ]						; then  USESPEKD3D47="1"; fi
 		if [ -z "$SDLUSEWAYLAND" ]						; then  SDLUSEWAYLAND="0"; fi
 		if [ -z "$STLRAD_PFTST" ]                       ; then  STLRAD_PFTST="none"; fi
+		if [ -z "$STEAMDECK_AUTOUP" ]                   ; then  STEAMDECK_AUTOUP="1"; fi
```

Comment:
Should we set this to `1` (enabled) by default? I don't remember if we wanted that behaviour or not, I'm not _super_ bothered either way but to me I would probably expect this to be off by default. However if you think differently please shout!

Note that if you test this in future and want to change this locally, you will need to remove the value from your `~/.config/steamtinkerlaunch/global.conf` and then bump the version. Values are only added when the version changes, and values are not automatically changed on version changes either; if we need to change a value after a release, we have `migrateCfgOption` to do so.

Reply:
Since this autoupdate feature is only for the Steam Deck it should be enabled by default because I see it like that :
we add a new feature that **requires** the new libs update, we want the users to automatically get the right lib if they update STL altogether. I think enabling it by default will prevent any breakages.

Reply:
> Agreed, although, **SteamTinkerLaunch is for power users only**, so everyone using it should know what they're doing, or they should be prepared to mess something up :-)

Yea that's why I put it in italic, but if we can avoid some bug reports and issues it's always a welcomed default !